src/styles/theme.ts

```diff
@@ -44,10 +56,31 @@ const theme = createTheme({
       main: red.A400,
     },
   },
+  breakpoints: {
+    values: {
+      // ? default breakpoints
+      // xs: 0,
+      // sm: 600,
+      // md: 900,
+      // lg: 1200,
```

Comment:
Why are we deviating from the default breakpoints?

Reply:
@amuqeet0

Reply:
I have added these values for demonstration purpose we can change the default breaking point
and more research needed to define the avg screen size/px for setting the mobile, tab and desktop screen as values provided by MUI desnt seem right